gremlinapi/attack_helpers.py

```diff
@@ -287,7 +319,22 @@ def __repr__(self):
                 model['hosts'] = {
                     'multiSelectTags': self.tags
                 }
-        return json.dumps(model)
+        return model
+
+    # def __repr__(self):
```

Comment:
The commented out blocks in this file can be removed unless you wanted to keep them around for another purpose.

Reply:
Yep, left them in for the time being as in case a future issue crops up where I may need to reference or revert